src/Config/packages.aget

```diff
@@ -0,0 +1,36 @@
+{
+  "nuget": {
+    "repos": [
+      "team-dynamo-nuget"
+    ],
+    "references": {
+      "DynamoVisualProgramming.Core" : "2.0.0-beta3402",
+      "DynamoVisualProgramming.DynamoCoreNodes" : "2.0.0-beta1",
+      "DynamoVisualProgramming.DynamoServices" : "2.0.0-beta3402",
+      "DynamoVisualProgramming.Tests" : "2.0.0-beta3402",
+      "DynamoVisualProgramming.WpfUILibrary" : "2.0.0-beta3402",
+      "DynamoVisualProgramming.ZeroTouchLibrary" : "2.0.0-beta3402",
+      "Greg" : "1.0.6176.18754",
+      "GregRevitAuth" : "1.0.5907.17451",
+      "HelixToolkit.Wpf.SharpDX" : "2015.1.629",
```

Comment:
As discussed, references to `HelixToolkit` and `SharpDX` can be removed from `RevitNodesUI.csproj`, hence we can remove these 12 packages from this `aget` specification.

Am I missing a place where `Cyotek.Drawing.BitmapFont` package gets used?